### ISSUE TITLE
Doesn't recognize `async def`

### ISSUE THREAD

**Issue by vad** [timothycrosley/isort#537]
In the following code:

```py
from myapp import myfunction

 
async def test_myfunction(test_client, app):
    a = await myfunction(test_client, app)
```

`isort` (4.2.5) tries to remove a newline. Are `async def` recognized by isort?

**Comment by timothycrosley** [comment]
This is fixed in the just released to PYPI `4.2.8` release.

Thanks!

~Timothy

**Comment by vad** [comment]
:+1: it works, thank you!

**Comment by meisenberg** [comment]
This is broken again in isort 4.2.15.

**Comment by timothycrosley** [comment]
@meisenberg I've added an additional test case with the exact usage asked about in this issue, and it passes against 4.2.15 - what is the exact case you are running into, as this is indeed fixed in 4.2.15?

Thanks!

~Timothy

**Comment by meisenberg** [comment]
Apologies, I can't reproduce now; I must have had a different version of the package installed than I thought.
